### PR TITLE
add grid layout to download links

### PR DESCRIPTION
**Pull request by noClaps**
Updated download links

Resolves: # 

- Added text to download links
- Now uses grid layout

### CODE REVIEW

File: docs/cloud.en.md [hunk 1]
```diff
@@ -22,15 +22,15 @@ If these alternatives do not fit your needs, we suggest you look into [Encryptio
...
-        [:fontawesome-brands-app-store-ios:](https://apps.apple.com/app/id1125420102){ .card-link title=App Store }
+        [:fontawesome-brands-windows: Windows](https://nextcloud.com/install/#install-clients){ .card-link title=Windows }
+        [:fontawesome-brands-apple: macOS](https://nextcloud.com/install/#install-clients){ .card-link title=macOS }
+        [:fontawesome-brands-linux: Linux](https://nextcloud.com/install/#install-clients){ .card-link title=Linux }
+        [:fontawesome-brands-freebsd: FreeBSD](https://www.freshports.org/www/nextcloud){ .card-link title=FreeBSD }
+        [:pg-openbsd: OpenBSD](https://openports.se/www/nextcloud){ .card-link title=OpenBSD }
+        [:pg-netbsd: NetBSD](https://pkgsrc.se/www/php-nextcloud){ .card-link title=NetBSD }
+        [:fontawesome-brands-google-play: Google Play](https://play.google.com/store/apps/details?id=com.nextcloud.client){ .card-link title="Google Play" }
+        [:pg-f-droid: F-Droid](https://f-droid.org/packages/com.nextcloud.client){ .card-link title=F-Droid }
+        [:fontawesome-brands-app-store-ios: App Store](https://apps.apple.com/app/id1125420102){ .card-link title=App Store }
```
Comment (jonaharagon) [inline docs/cloud.en.md:33]:
```suggestion
        [:fontawesome-brands-windows: Windows](https://nextcloud.com/install/#install-clients)
        [:fontawesome-brands-apple: macOS](https://nextcloud.com/install/#install-clients)
        [:fontawesome-brands-linux: Linux](https://nextcloud.com/install/#install-clients)
        [:fontawesome-brands-freebsd: FreeBSD](https://www.freshports.org/www/nextcloud)
        [:pg-openbsd: OpenBSD](https://openports.se/www/nextcloud)
        [:pg-netbsd: NetBSD](https://pkgsrc.se/www/php-nextcloud)
        [:fontawesome-brands-google-play: Google Play](https://play.google.com/store/apps/details?id=com.nextcloud.client)
        [:pg-f-droid: F-Droid](https://f-droid.org/packages/com.nextcloud.client)
        [:fontawesome-brands-app-store-ios: App Store](https://apps.apple.com/app/id1125420102)
```

File: docs/cloud.en.md [hunk 2]
```diff
@@ -88,9 +88,9 @@ When using a web client, you are placing trust in the server to send you proper
...
-        [:fontawesome-brands-apple:](https://tahoe-lafs.readthedocs.io/en/latest/Installation/install-tahoe.html#linux-bsd-or-macos){ .card-link title=macOS }
-        [:fontawesome-brands-linux:](https://tahoe-lafs.readthedocs.io/en/latest/Installation/install-tahoe.html#linux-bsd-or-macos){ .card-link title=Linux }
-        [:pg-netbsd:](https://tahoe-lafs.readthedocs.io/en/latest/Installation/install-tahoe.html#linux-bsd-or-macos){ .card-link title=NetBSD }
+        [:fontawesome-brands-windows: Windows](https://tahoe-lafs.readthedocs.io/en/latest/Installation/install-tahoe.html#microsoft-windows){ .card-link title=Windows }
+        [:fontawesome-brands-apple: macOS](https://tahoe-lafs.readthedocs.io/en/latest/Installation/install-tahoe.html#linux-bsd-or-macos){ .card-link title=macOS }
+        [:fontawesome-brands-linux: Linux](https://tahoe-lafs.readthedocs.io/en/latest/Installation/install-tahoe.html#linux-bsd-or-macos){ .card-link title=Linux }
+        [:pg-netbsd: NetBSD](https://tahoe-lafs.readthedocs.io/en/latest/Installation/install-tahoe.html#linux-bsd-or-macos){ .card-link title=NetBSD }
```
Comment (jonaharagon) [inline docs/cloud.en.md:94]:
```suggestion
        [:fontawesome-brands-windows: Windows](https://tahoe-lafs.readthedocs.io/en/latest/Installation/install-tahoe.html#microsoft-windows)
        [:fontawesome-brands-apple: macOS](https://tahoe-lafs.readthedocs.io/en/latest/Installation/install-tahoe.html#linux-bsd-or-macos)
        [:fontawesome-brands-linux: Linux](https://tahoe-lafs.readthedocs.io/en/latest/Installation/install-tahoe.html#linux-bsd-or-macos)
        [:pg-netbsd: NetBSD](https://tahoe-lafs.readthedocs.io/en/latest/Installation/install-tahoe.html#linux-bsd-or-macos)
```

**Review by jonaharagon — CHANGES_REQUESTED** [review]
Plus we'll have to go through and do the rest of the pages.

File: docs/cloud.en.md (hunk 1 above)
Comment (noClaps) [reply docs/cloud.en.md:33]: I think you'll need the `{.card-link}` since the CSS refers to it. The `title=" "` can be removed though.

**Comment by dngray** [comment]
You know you can sync, https://help.github.com/articles/syncing-a-fork/

**Comment by noClaps** [comment]
I tried that, but it kept giving me errors and not working, so after a while I just gave up and re-did the whole thing. Sorry, I'm not very good with git and working with forks.

**Comment by dngray** [comment]
i tend to find i have less issues with `git rebase main` than `git merge`. Rebase "re-plays" your changes on top of the main repo.

Of course if there's been major changes you'll get merge conflicts that need to be resolved first, before you can `git rebase --continue`.

**Comment by noClaps** [comment]
The thing is, after my changes, the entire directory structure was changed, and the files I was working on were moved. For instance, `extra.css` was moved from `docs/stylesheets/extra.css` to `docs/assets/stylesheets/extra.css`, so doing `git rebase` wouldn't help since the file on which I'm "re-playing" changes doesn't exist at that location anymore.

Honestly, there probably was a way I could have done it, but remaking the fork and PR just seemed a lot easier.

**Pull request closed without merging**